src/nunit-gui/Presenters/NUnitTreeDisplayStrategy.cs

```diff
@@ -57,14 +57,21 @@ public override void OnTestLoaded(TestNode testNode)
         {
             ClearTree();
 
+            TreeNode topNode = null;
             foreach (var topLevelNode in testNode.Children)
             {
                 var treeNode = MakeTreeNode(topLevelNode, true);
 
+                if (topNode == null)
+                    topNode = treeNode;
+
                 _view.Tree.Add(treeNode);
 
                 SetInitialExpansion(treeNode);
             }
+
+            if (topNode != null)
+                topNode.EnsureVisible();
```

Comment:
It's early days, but I'll give it a try. We really need to formulate some sort of standard about how agressively we want to use this and other features.